mozilla-release/configure.in

```diff
@@ -8581,7 +8581,6 @@ AC_SUBST(MOZ_CRASHREPORTER)
 AC_SUBST(MOZ_CRASHREPORTER_INJECTOR)
 AC_SUBST(MOZ_CRASHREPORTER_UPLOAD_FULL_SYMBOLS)
 AC_SUBST(MOZ_MAINTENANCE_SERVICE)
-AC_SUBST(MOZ_STUB_INSTALLER)
```

Comment:
can this be commented with appropriate comment? may be hard to find otherwise when we will want to get it back